src/toolbar_gui.cpp

```diff
@@ -147,9 +147,11 @@ static void PopupMainToolbarMenu(Window *w, WidgetID widget, const std::initiali
 }
 
 /** Enum for the Company Toolbar's network related buttons */
-static const int CTMN_CLIENT_LIST = -1; ///< Show the client list
-static const int CTMN_SPECTATE    = -2; ///< Become spectator
-static const int CTMN_SPECTATOR   = -3; ///< Show a company window as spectator
+enum class CompanyToolbarMenu : int {
+	ClientList = -1, ///< Show the client list
```

Comment:
I don't see a particular need to keep the negative constants, just let enum do its thing (unless there's a good reason, in which case it should be documented)

Reply:
These are for a company list dropdown, so if not negative they'd need to be some value >= MAX_COMPANIES.

Reply:
```suggestion
	ClientList = MAX_COMPANIES, ///< Show the client list
```
then :) (with a comment)